### DISCUSSION TITLE
Scomparsa Hacs e problema dashboard

### DISCUSSION THREAD

**Discussion by gpjob** [indomus/forum#3245] [HUB: Home Assistant]
Salve e perdonatemi ma sto impazzendo, 
da qualche giorno, all'apertura della mia dashboard mi appare il seguente errore:

` ! Custom element doesn't exist: masonry-layout`
con sfondo rosso e tutte le righe di codice
Non appare nemmeno la scritta "Hacs" sul menu' a sinistra e nel file di registro trovo il seguente errore:

`Registratore: homeassistant.setup
Fonte: setup.py:363
Prima occorrenza: 09:54:09 (1 occorrenze)
Ultima registrazione: 09:54:09
The 'hacs' integration does not support YAML setup, please remove it from your configuration`

Ho provato a caricare un backup di circa 2 settimane prima e funziona tutto, allora ho pensato che siano gli aggiornamenti ed, infatti, rifacendoli, si e' ricreato l'errore.

Ma io nel configuration.yaml ho solo una riga di codice con il token hacs e nient'altro... 

Preciso che un'altra dashboard senza mushroom (utilizzate nella prima) e quindi ricorso ad Hacs, funziona perfettamente...

Secondo voi, cosa e'?

Grazie mille anticipato

**Comment by Scialla** [comment]
quella riga di codice che hai, devi toglierla. Ora HACS si configura via GUI

**Reply by gpjob** [reply]
Fatto, riavviato, ma rimane errore e ancora non si vede Hacs...

**Reply by Scialla** [reply]
Se rimane l'errore `'hacs' integration does not support YAML setup, please remove it from your configuration`, che è ben chiaro e ti dice di rimuovere la configurazione HACS, vuol dire che non hai solo quella riga..

**Reply by gpjob** [reply]
Intendi solo nel configuration.yaml?

**Reply by Scialla** [reply]
Non sono io che intendo... l'errore è chiaro! Dove hai configurato HACS lo sai tu, di solito si mette lì, poi se tu hai inserito la configurazione in package lo sai tu, cerca

**Reply by gpjob** [reply]
perdonami, sono veramente alle prime armi: ho visto che otlre alle due righe nel configuration.yaml ho nella cartella config > custom_components una cartella hacs... allora dovrebbe essere quella?
credo si sia creata quando ho provato a fare dashboard con mushroom card, che, appunto, non funzionano piu'...

**Reply by Scialla** [reply]
No, quello è HACS e non la sua configurazione. Tu devi rimuovere le righe di configurazione che, di solito, stanno nel configuration.yaml. Se continua a darti l'errore non le hai rimosse.

https://indomus.it/guide/installare-hacs-home-assistant-community-store-sul-proprio-hub/
